px.py

```diff
@@ -775,12 +813,26 @@ def do_PATCH(self):
     def do_CONNECT(self):
         dprint("Entering")
 
+        # for CONNECT, just scrap connection: close
+        h_connection = ""
+        if self.headers["connection"] != None:
+            h_connection = self.headers["connection"]
+            del self.headers["connection"]
+            dprint("removed 'connection'")
+        if self.headers["Connection"] != None:
+            h_connection = self.headers["Connection"]
+            del self.headers["Connection"]
+            dprint("removed 'Connection'")
+
```

Comment:
Is this really required for do_CONNECT to work or can we limit the change to auth only?